documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +101,45 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is a json string which consist of all broker IDs and provides a number of metrics for each broker.
```

Comment:
"The broker load is stored in a ConfigMap (with the same name as the KafakRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker."

Reply:
Should we also include an example command of how to extract the JSON string from the config map using `kubectl`? 

WDYT @laidan6000?

Reply:
Sounds like that would be useful.

To extract the JSON string from the ConfigMap, use the following command:

Reply:
@ShubhamRwt can you add the `kubectl` command to extract the json?

Reply:
just the json right ? ```kubectl get configmap my-rebalance -o json```